themes/default/templates/header.mtt

```diff
@@ -0,0 +1,47 @@
+<nav class="section nav dark">
```

Comment:
It would be nice to also have a flag to disable the menu, since this is also very specific. Maybe we can add another flag to add a custom logo.